lib/matchers/hashmap.js

```diff
@@ -26,7 +26,7 @@ module.exports = factory({
 
     var errors = [];
     if (this.matchers.keys) {
-      var keyErrors = new s.array({of: this.matchers.keys}).match(path + '.keys', Object.keys(obj));
+      var keyErrors = s.array({of: this.matchers.keys}).match(path + '.keys', Object.keys(obj));
```

Comment:
is there a reason we don't have a json schema for hashMap
would it be similar to the object one?